### PR TITLE
Release/thidwick to develop

### PR DESCRIPTION
**Pull request by philtom-ctds**

### CODE REVIEW

File: gdcdictionary/schemas/somatic_copy_number_workflow.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ links:
       - name: submitted_genotyping_arrays
         backref: somatic_copy_number_workflows
         label: performed_on
-        target_type: submitted_genotyping_arrays
+        target_type: submitted_genotyping_array
```
Comment (ProfOak) [inline gdcdictionary/schemas/somatic_copy_number_workflow.yaml]: `target_type` keys are normally pluralized. Why did we opt for making this singular? It looks like we don't have any of this node in the system so it won't break anything existing, but I'm still curious.

Comment (anmaxvl) [reply gdcdictionary/schemas/somatic_copy_number_workflow.yaml]: I think link `name` is pluralized, but not the `target_type`.

Comment (philtom-ctds) [reply gdcdictionary/schemas/somatic_copy_number_workflow.yaml]: @ganjigajanan @wwysoc2 Either of you able to answer the question above?

Comment (ganjigajanan) [reply gdcdictionary/schemas/somatic_copy_number_workflow.yaml]: @ProfOak, Link `name` & `backref` were always pluralized  & `target_type` refers to the node_type and hence needs to be singular. e.g. `aligned_reads` seems to appear plural but in dictionary context it is the name of the node.

Comment (ProfOak) [reply gdcdictionary/schemas/somatic_copy_number_workflow.yaml]: Ah ok my mistake. Thanks for jumping in and answering everyone.

**Pull request merged**